torchci/pages/reliability/[repoOwner]/[repoName]/[[...page]].tsx

```diff
@@ -43,28 +49,87 @@ const URL_PREFIX = `/reliability/pytorch/pytorch?jobName=`;
 function GroupReliabilityPanel({
   title,
   queryName,
+  queryCollection,
   queryParams,
   metricHeaderName,
   metricName,
   filter,
 }: {
   title: string;
   queryName: string;
+  queryCollection: string;
   queryParams: RocksetParam[];
   metricHeaderName: string;
   metricName: string;
   filter: any;
 }) {
+  const url = `/api/query/${queryCollection}/${queryName}?parameters=${encodeURIComponent(
+    JSON.stringify(queryParams)
+  )}`;
+
+  const { data } = useSWR(url, fetcher, {
+    refreshInterval: 60 * 60 * 1000,
```

Comment:
Is this supposed to refresh once an hour?

Reply:
Yeah, this doesn't need to be refreshed that often.  In fact, I don't want the page to be refreshed when I'm looking at it, so I kind of set a very high value here :)